releasenotes/notes/deprecate_clifford_indexing-5e3500301a696bdc.yaml

```
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Remove deprecated getitem/setitem magic methods of 
+    :class:`~qiskit.quantum_info.operators.symplectic.Clifford`.
```

Traditionally, we explain the alternative path in the release note too
```suggestion
    Remove deprecated ``__getitem__``/``__setitem__`` magic methods of 
    :class:`~qiskit.quantum_info.operators.symplectic.Clifford`.
    The methods were deprecated since Qiskit 0.44, released on July 2023. Instead, index or iterate through the :attr:`.Clifford.tableau` attribute.
```